test/CMakeLists.txt

```diff
@@ -28,6 +22,17 @@ if(LIBCORO_FEATURE_NETWORKING)
     )
 endif()
 
+if(LIBCORO_FEATURE_THREADING)
```

Comment:
What would you consider a good name for this option variable moving forward? `THREADING` doesn't quite seem to make sense to me anymore but I'm not sure what it _should_ be.  Do you have any suggestions?

Reply:
Let me think a bit on this...

Reply:
I still like `THREADING` -- at its heart, this is the `io_scheduler.hpp/cpp` that schedules coroutines across multiple threads; everything else (from what I see) supports this. The items moved outside of `THREADING` don't belong in threading, since they support coroutines, not threads.

Reply:
Counterpoint: You can run `io_scheduler` on 1 thread and inline coroutine execution on the `io_scheduler`'s event loop thread instead of having a `coro::thread_pool` to execute ready coroutines.  I also think the _threading_ part isn't the issue with cross platform, its the kernel events (e.g. epoll) that are the real problem right now since its basically hardcoded to linux epoll where as `std::thread` should be portable.  Something like... `LIBCORO_FEATURE_IO` might make sense?

Reply:
That's true, it's the epoll dependency that causes problems.

Reply:
https://github.com/jbaldwin/libcoro/issues/183